src/index.html

```diff
@@ -1,13 +1,53 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
-</html>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./Styles/index.css">
+  </head>
+
+  <body>
+    <div class="card" data-qa="card">
+
+      <img 
+        class="card__image" 
+        src="./images/imac.jpeg" 
+        alt="product image"
+      >
+
+      <h2 class="card__product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>  
+      <p class="card__product-code">Product code: 195434</p>
+
+      <div class="card__reviews">
+        <div class="stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+        </div>
+        <p class="card__reviews-no">Reviews: 5</p>
+      </div>
+
+      <div class="card__price">
+        <p class="card__price-title">Price:</p>
+        <h3 class="card__price-value">$2,199</h3>
+      </div>
+
+      <a 
+        class="card__buy-cta" 
+        href="#" 
+        data-qa="hover"
+      >
+        Buy
+      </a>
+
+    </div>
+  </body>
+
+</html>
```

Comment:
```suggestion
</html>

```
don't forget to add empty line in the end of the file